posthog/models/team.py

```diff
@@ -88,7 +88,9 @@ class Team(models.Model):
     ingested_event: models.BooleanField = models.BooleanField(default=False)
     uuid: models.UUIDField = models.UUIDField(default=UUIDT, editable=False, unique=True)
     session_recording_opt_in: models.BooleanField = models.BooleanField(default=False)
-    session_recording_retention_period_days: models.IntegerField = models.IntegerField(null=True, default=None)
+    session_recording_retention_period_days: models.IntegerField = models.IntegerField(
+        null=True, default=None, blank=True
```

Comment:
Unrelated change? Also doesn't blank mean empty string? This value is an integer so this seems wrong.

Reply:
It's only a validation thing, we weren't able to edit organizations in Django admin otherwise. In fact blank is not string related at all, it only means that the field can be left empty (in this case null). Otherwise Django admin wouldn't save the org edit without providing a number in this field. The same applies to these Team fields updated here.